app/policies/idv/flow_policy.rb

```diff
@@ -2,15 +2,54 @@
 
 module Idv
   class FlowPolicy
-    attr_reader :idv_session, :user
+    attr_reader :analytics, :idv_session, :user
 
     FINAL = :final
 
-    def initialize(idv_session:, user:)
+    def initialize(analytics:, idv_session:, user:)
+      @analytics = analytics
       @idv_session = idv_session
       @user = user
     end
 
+    def steps
```

Comment:
Changes to this method:
1. Made it public (so I could use it in a spec)
2. Added `@steps ||=` to the beginning so steps are cached
3. Added `.freeze` to the object definition

Reply:
what's the purpose of the freeze? was code modifying this sneakily?